vptq/app.py

```diff
@@ -8,7 +8,7 @@
 import gradio as gr
 
 from vptq.app_utils import get_chat_loop_generator
-
+from app_gpu import update_charts as _update_charts
```

Comment:
I hit a error "ModuleNotFoundError: No module named 'app_gpu'" in running the command " python -m vptq.app "

Should we use `from vptq.app_gpu xxxxx` ?

Reply:
yeah, agree with it

Reply:
Thanks again for your code review

Reply:
just in case, I installed VPTQ through `pip install -e .`, and the change will synchronize to the module path. If VPTQ is installed through remote git, it needs re-install. otherwise it can't be find at `PYTHONPATH`.

tested at random directory.
```shell
(base) root@a100:/data# python -m vptq.app
Replacing linear layers...: 100%|████████████████████████████████████████████████████████████████████████████████████████| 399/399 [00:00<00:00, 1878.59it/s]
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
```

Reply:
`pip install git+https://github.com/TITC/VPTQ.git --no-build-isolation` passed for me.